_pages/en/strategy-build-vs-buy.md

```diff
@@ -37,66 +44,140 @@ th {
+      - IT Solutions Architecture is a key partner throughout the lifecycle of the IT Solutions but IT Products teams have authority to operate and take responsibility for their own products.
+      - IT Solutions are architected iteratively over time, with each gate and iteration providing additional information as to choosing the best IT Products to compose them (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html)).
+      - Leveraging external third party integrators is supported but IT Solutions Architecture remains in control to ESDC throughout its lifecycle and abides by its [Architectural Principles](#architecture).
+   2. Complex but not core to mandate IT Solutions are designed internally.
+   3. Non-complex and not core to mandate IT Solutions may be designed and delivered by third party integrators.
+3. IT Solutions, whether built internally or built by a third party and delivered as a set of pre-packaged COTS to configure and/or as software as a service, adhere to the Architectural principles as listed in section [Architecture](#architecture)
+4. IT Solutions must provide their overall and individual IT products metrics internally to ESDC. Metrics are comprised of the following:
+   - Accessibility test results
+   - Integration testing results
+   - Security test results
+   - API consumption statistics
+   - Production deployment audit traces
+   - Availability statistics
+5. Security Assessment & Authority (SA&A) uses the Target-SA&A methodology
+6. Accessibility Assessment uses the Target-Accessibility methodology
+7. Audit Assessment uses the Target-Audit methodology
+8. IT Solutions reuse existing IT Products, including:
+    - Internally developed software
+    - Open Source Software (internally and externally developed)
+    - Third party Services (SaaS, PaaS, and IaaS) and COTS
+
+### Finance
+
+1. IT Solutions sizes are minimized as much as possible
+2. IT Solutions leverage existing, reusable, scalable and approved IT Products to minimize cost when possible
+   - IT Products requiring purchased licences are not deemed as reusable or scalable for the purpose of this criterion.
+3. IT Solutions, once in production, have dedicated and recurring budgets to support their evolution over their lifecycle, and are managed in tandem by the primary business owner and the dedicated IT Products teams.
+   - The IT Products teams are responsible for maintaining their own products as part of the IT Solutions. (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html))
+   - A same team managing an IT Product may provide service to multiple IT Solutions; IT Products are thus required to have a scalable and easily adaptable financial model supporting multiple business owners if required (similar to SaaS pay as you use or yearly subscriptions per users, etc.).
+
+### Architecture
```

Comment:
Should provide more detailed on the rules to follow. In my mind:
- Loose Coupling boundaries are at the Business Capability Model level. So duplication of IT Products or IT Solutions may be acceptable as long as they are for different business capabilities. This to provide more autonomy to teams and provides the necessary agility for ESDC to change
- IT Solutions are comprised of IT Products, which are comprised of Software+configuration, which are deployed on Infrastructure+configuration. An IT solution definition needs to encompass all these
- Vendor locking risk needs to be properly assessed (should probably provide a framework to do so - as part of the action plan)

Reply:
Yes to all the above but especially providing tools to assess vendor lock in risk as part of the action plan.  Would it make sense to have a 'total cost of ownership' assessment framework? It would be broader than just vendor lock in risk....would provide guidance to assess the capital costs (e.g. purchase costs), O&M costs (development costs,  maintenance costs (e.g. annual licence maintenance, OSS maintenance), SAL costs (for in house development, ongoing support) and 'lock in costs' (e.g. if you want out 10 years from now)

Reply:
I've added a Lock-in Assessment Framework as a TODO item. Lock-In is not only at the vendor level (although it's one of the worst IMHO) as it could be technological as well.

Reply:
Now thinking this should be part of the risk assessment framework as a section...

Reply:
Created two different risk assessment frameworks as actions to do: IT Solutions AND IT products